server/athenian/api/invite_admin.py

```diff
@@ -15,14 +15,30 @@ def main():
     engine = create_engine(sys.argv[1])
     session = sessionmaker(bind=engine)()
     salt = randint(0, (1 << 16) - 1)
-    if not session.query(Account).filter(Account.id == invitation_controller.admin_backdoor).all():
+    admin_backdoor = invitation_controller.admin_backdoor
+    if not session.query(Account).filter(Account.id == admin_backdoor).all():
         session.add(Account(id=invitation_controller.admin_backdoor))
         session.commit()
+        max_id = session.query(func.max(Account.id)).filter(Account.id < admin_backdoor).first()
+        if max_id is None or max_id[0] is None:
+            max_id = 0
+        else:
+            max_id = max_id[0]
+        max_id += 1
+        if engine.url.drivername == "postgres":
+            engine.execute("ALTER SEQUENCE accounts_id_seq RESTART WITH %d;" % max_id)
```

Comment:
@se7entyse7en This is the fix - I tested with both sqlite and postgres, and I learned something new about AUTOINCREMENT :D SQLite has primary keys autoincremented by default, but the counter values are calculated dynamically as `MAX(id)`. This is not what we want, so I had to force SQLite `AUTOINCREMENT` keyword feature, hence `sqlite_autoincrement=True`. Postgres does not have this quirk so it was easier to deal with.

Reply:
Thanks!